sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -43,7 +70,7 @@ public async Task AnalyzeOperationTest()
 
             AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchConvenienceDocuments, operationOptions, "en");
 
-            await operation.WaitForCompletionAsync();
+            await operation.WaitForCompletionAsync(PollingInterval);
```

Comment:
Do we need to rerecord the tests with the new polling interval?

Reply:
Because this should only affect playback mode. re-recording shouldn't be necessary.

Reply:
I think I was tired yesterday.